packages/flutter_tools/test/integration.shard/downgrade_upgrade_integration_test.dart

```diff
@@ -50,43 +50,50 @@ void main() {
     final Directory testDirectory = parentDirectory.childDirectory('flutter');
     testDirectory.createSync(recursive: true);
 
+    int exitCode = 0;
+
     // Enable longpaths for windows integration test.
     await processManager.run(<String>[
       'git', 'config', '--system', 'core.longpaths', 'true',
     ]);
 
     // Step 1. Clone the dev branch of flutter into the test directory.
-    await processUtils.stream(<String>[
+    exitCode = await processUtils.stream(<String>[
       'git',
       'clone',
       'https://github.com/flutter/flutter.git',
     ], workingDirectory: parentDirectory.path, trace: true);
+    expect(exitCode, 0);
 
     // Step 2. Switch to the dev branch.
```

Comment:
These comments refer to dev branch anyway :)

Reply:
Ahh ... yes